src/curated-corpus/components/ApprovedItemCardWrapper/ApprovedItemCardWrapper.test.tsx

```diff
@@ -4,19 +4,23 @@ import { MemoryRouter } from 'react-router-dom';
 import { ApprovedCorpusItem } from '../../../api/generatedTypes';
 import { ApprovedItemCardWrapper } from './ApprovedItemCardWrapper';
 import { getTestApprovedItem } from '../../helpers/approvedItem';
+import { ThemeProvider } from '@mui/material/styles';
+import theme from '../../../theme';
 
 describe('The ApprovedItemCardWrapper component', () => {
   const item: ApprovedCorpusItem = getTestApprovedItem();
 
   it('should render an approved item card', () => {
     render(
       <MemoryRouter>
-        <ApprovedItemCardWrapper
-          item={item}
-          onEdit={jest.fn()}
-          onReject={jest.fn()}
-          onSchedule={jest.fn()}
-        />
+        <ThemeProvider theme={theme}>
+          <ApprovedItemCardWrapper
+            item={item}
```

Comment:
There are multiple fixes to tests along the same lines. Components need to be wrapped within a `ThemeProvider` and provided our app's theme because somewhere deep within there's a styled component that is using that theme to get the colour or tweak some styles for small screens, etc.